docs/src/pages/guides/FHIRSearchConfiguration.md

```diff
@@ -263,20 +263,43 @@ In order to avoid this issue, inclusion criteria search parameters should not be
+|`tstamp`|string|Reindex only resources not previously reindexed since this timestamp. Format as a date YYYY-MM-DD or time YYYY-MM-DDTHH:MM:SSZ.|
 |`resourceCount`|integer|The maximum number of resources to reindex in this call. If this number is too large, the processing time might exceed the transaction timeout and fail.|
 
 The IBM FHIR Server tracks when a resource was last reindexed and only resources with a reindex_tstamp value less than the given tstamp parameter will be processed. When a resource is reindexed, its reindex_tstamp is set to the given tstamp value. In most cases, using the current date (for example "2020-10-27") is the best option for this value.
 
-To aid in the re-indexing process, the IBM FHIR Server team has expanded the fhir-bucket resource-loading tool to support driving the reindex. The fhir-bucket tool uses a thread-pool to make concurrent POST requests to the IBM FHIR Server `$reindex` custom operation.
+### 2.2 Client-side-driven approach
+Another option is to have the client utilize the `$retrieve-index` and ``$reindex` in parallel to drive the processing.
 
-For more information on driving the reindex operation from fhir-bucket, see https://github.com/IBM/FHIR/tree/main/fhir-bucket#driving-the-reindex-custom-operation.
+The `$retrieve-index` operation is called to retrieve logical resource IDs of resources available for reindexing. This can be done repeated by using the `_count` and `afterLogicalResourceId` parameters for pagination. The operation supports the following parameters to control the behavior:
+
+|name|type|description|
+|----|----|-----------|
+|`_count`|string|The maximum number of logical resource IDs to retrieve. This may not exceed 1000. If not specified, the maxinum number retrieved is 1000.|
+|`notModifiedAfter`|string|Only retrieve logical resource IDs for resources not last updated after this timestamp. Format as a date YYYY-MM-DD or time YYYY-MM-DDTHH:MM:SSZ.|
+|`afterLogicalResourceId`|string|Retrieve logical resource IDs starting with the first logical resource ID after this logical resource ID. If this parameter is not specified, the retrieved logical resource IDs start with the first logical resource ID.|
+
+The `$reindex` operation is called to reindex the resources with logical resource IDs in the specified list. The operation supports the following parameters to control the behavior:
 
-Reindexing is resource-intensive and can take several hours or even days to complete depending on the number of resources currently in the system and the capability of the hosting platform.
+|name|type|description|
+|----|----|-----------|
+|`logicalResourceIds`|string|Reindex only resources with a logical resource ID in the specified list, formatted as a comma-delimited list of strings. If number of logical resource IDs in the list is too large, the processing time might exceed the transaction timeout and fail.|
```

Comment:
is this our database-internal ids or the resource ids?
if the former, we need to change the name... "logical resource id" has a specific meaning to FHIR users and this aint it.

Reply:
I do realize that this is what its called in our database (as documented at https://github.com/IBM/FHIR/blob/main/fhir-persistence-schema/docs/SchemaMigration.md#logical_resources-and-resources) but I really don't want to expose this to our users because it is really confusing.  Happy to brainstorm alternative names as needed.

Reply:
I could just change the name to something like "indexId", so users just see it as just some ID used in reindex processing.

Reply:
Updated to use the name "indexIds" for the parameter, instead of "logicalResourceIds".  This should avoid confusing user into thinking it is the same as resource IDs.

Reply:
I am happy with `indexIds`...this is WAY better than `logicalResourceIds` :-)